README.md

```diff
@@ -148,7 +148,8 @@ apl
 <br> Dumps info from tplg binary file.
 
 * tplgtool.py
-<br> Dumps info from tplg binary file. (deprecated)
+<br> Dumps info from tplg binary file. sof-tplgreader.py still use this but new features
+     will go to tplgtool2.py. Will be deprecated.
```

Comment:
Can you please explain a little bit more which script uses which script and when?

Reply:
The way this 3 scripts interact with each other is really not intuitive and it took me an unreasonable amount time to figure out. Since you're adding a new feature to the deprecated script (while leaving the newer script behind?), please add a couple more sentences describing the situation better. Right now this part of the README.md is worse than nothing.

Reply:
> The way this 3 scripts interact with each other is really not intuitive.
I agree this part.

I don't understand rest of your comment. Why do you think tplgtool.py is deprecated? README was misleading. When tplgtool2.py was introduced, (if it wants to replace tplgtool.py) it has to take care of all the functions in tplgtool.py. But it didn't. More likely it added a new file and put a deprecated notice on the currently being used script.

Reply:
> Why do you think tplgtool.py is deprecated? 

Because it says so:
 
```
 warnings.warn("tplgtool.py is deprecated, use tplgtool2.py instead.", DeprecationWarning)

```

Reply:
Glad I'm not the only one confused!